src/main/java/org/isf/accounting/rest/BillController.java

```diff
@@ -0,0 +1,449 @@
+package org.isf.accounting.rest;
+
+import static org.isf.shared.mapper.OHModelMapper.getObjectMapper;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.GregorianCalendar;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.isf.accounting.dto.BillDTO;
+import org.isf.accounting.dto.BillItemsDTO;
+import org.isf.accounting.dto.BillPaymentsDTO;
+import org.isf.accounting.dto.FullBillDTO;
+import org.isf.accounting.manager.BillBrowserManager;
+import org.isf.accounting.model.Bill;
+import org.isf.accounting.model.BillItems;
+import org.isf.accounting.model.BillPayments;
+import org.isf.patient.dto.PatientDTO;
+import org.isf.patient.manager.PatientBrowserManager;
+import org.isf.patient.model.Patient;
+import org.isf.priceslist.manager.PriceListManager;
+import org.isf.priceslist.model.PriceList;
+import org.isf.shared.exceptions.OHAPIException;
+import org.isf.utils.exception.OHServiceException;
+import org.isf.utils.exception.model.OHExceptionMessage;
+import org.isf.utils.exception.model.OHSeverityLevel;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.Authorization;
+
+
+@RestController
+@Api(value="/bills",produces = MediaType.APPLICATION_JSON_VALUE, authorizations = {@Authorization(value="basicAuth")})
+public class BillController {
+	
+	@Autowired
+	protected BillBrowserManager billManager;
+	
+	@Autowired
+	protected PriceListManager priceListManager;
+	
+	@Autowired
+	protected PatientBrowserManager patientManager;
+
+	private final Logger logger = LoggerFactory.getLogger(BillController.class);
+
+	public BillController(BillBrowserManager billManager) {
+		this.billManager = billManager;
```

Comment:
This constructor is not needed as the manager is autowired.

Reply:
OK It's right. I am correcting it now.